_shared/project/tox.ini

```diff
@@ -25,13 +25,22 @@ skip_install =
 setenv =
     PYTHONUNBUFFERED = 1
     OBJC_DISABLE_INITIALIZE_FORK_SAFETY = YES
+    dev: DEV = {env:DEV:true}
+    dev: SENTRY_ENVIRONMENT = {env:SENTRY_ENVIRONMENT:dev}
+    dev: NEW_RELIC_APP_NAME = {env:NEW_RELIC_APP_NAME:{{ cookiecutter.slug }}}
+    dev: NEW_RELIC_ENVIRONMENT = {env:NEW_RELIC_ENVIRONMENT:dev}
 {% if cookiecutter.get("postgres") == "yes" %}
+    SQLALCHEMY_SILENCE_UBER_WARNING=1
```

Comment:
This envvar has been added to a few of our projects so I added it to the cookiecutter